pkg/collector/corechecks/snmp/values.go

```diff
@@ -26,7 +26,7 @@ func (v *resultValueStore) getScalarValue(oid string) (snmpValueType, error) {
 func (v *resultValueStore) getColumnValues(oid string) (map[string]snmpValueType, error) {
 	values, ok := v.columnValues[oid]
 	if !ok {
-		return nil, fmt.Errorf("value for Column OID `%s` not found in `%v`", oid, v.columnValues)
+		return nil, fmt.Errorf("value for Column OID `%s` not found in results", oid)
```

Comment:
what about adding a debug message? If there is a way for the message itself to be evaluated only when debug mode is enabled

Reply:
@pducolin 
The error is currently logged as debug here: https://github.com/DataDog/datadog-agent/blob/ca43b78a773c2a470cb9c019491e363228a39614/pkg/collector/corechecks/snmp/config_metric.go#L97-L101

But the message itself is not that useful too, the same information can be retrieved from either request/reponse call debug logs.

Or via the whole valuesStore: https://github.com/DataDog/datadog-agent/blob/6ba7fc4f5cf5187ea7c6d61a26b22ddb71863e17/pkg/collector/corechecks/snmp/snmp.go#L100